tests/system/large/test_remote_function.py

```diff
@@ -1300,3 +1300,34 @@ def square_num(x):
         cleanup_remote_function_assets(
             session.bqclient, session.cloudfunctionsclient, square_num
         )
+
+
+@pytest.mark.parametrize(
+    ("max_batching_rows"),
+    [
+        10_000,
+        None,
+    ],
+)
+@pytest.mark.flaky(retries=2, delay=120)
+def test_remote_function_max_batching_rows(session, scalars_dfs, max_batching_rows):
```

Comment:
Would be good if we can verify the config of the created function.

Reply:
makes sens, done, thanks.